ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -3,7 +3,7 @@ yum install -y jq && \
 bash -c ' \
   if [ -n "__GANESHA_PACKAGES__" ]; then \
     if [[ "${CEPH_VERSION}" == master || "${CEPH_VERSION}" == main ]]; then \
-      curl -s -L "https://shaman.ceph.com/api/repos/nfs-ganesha/V5.3.2/latest/centos/__ENV_[DISTRO_VERSION]__/flavors/ceph_main/repo?arch=$(arch)" -o /etc/yum.repos.d/ganesha.repo ; \
+      curl -s -L "https://shaman.ceph.com/api/repos/nfs-ganesha/V5.5/latest/centos/__ENV_[DISTRO_VERSION]__/flavors/ceph_main/repo?arch=$(arch)" -o /etc/yum.repos.d/ganesha.repo ; \
```

Comment:
@ljflores I think we should test ceph devel builds with nfs-ganesha builds

Reply:
@kalebskeithley thoughts?

@guits in that case, we'd have to add something to the nfs-ganesha Jenkins job so it periodically builds "next" as well as V5.5: https://github.com/ceph/ceph-build/blob/main/nfs-ganesha/config/definitions/nfs-ganesha.yml

Reply:
'next' is the bleeding edge devel branch. It's not really suitable, IMO, to be used. For anything

If you use a 'next' branch build and something doesn't work we may or may not make it a priority to fix.

I strongly suggest you use only released/tagged VX.Y versions, e.g. V5.5, V5.6, etc.   V5.Y is perfectly fine for pacific, quincy, reef, and beyond.

I don't understand the comment that you need to build 'next' occasionally.  Why?

Reply:
In that case, let's stick with V5.5.